src/olympia/users/forms.py

```diff
@@ -133,6 +133,12 @@ def clean_display_name(self):
         return name
 
 
+USER_EDIT_EXCLUDE_FIELDS = [
+    'password', 'email', 'picture_type', 'last_login', 'fxa_id',
+    'read_dev_agreement',
+]
```

Comment:
I don't get why it should be a tuple. Semantically it is a list of fields and the positions mean nothing. It is a tuple in some other places though.